Tools/SandboxTest.ps1

```diff
@@ -212,14 +219,29 @@ function Get-RemoteContent {
     )
     Write-Debug "Attempting to fetch content from $URL"
     # Check if the URL is valid before trying to download
-    $response = [String]::IsNullOrWhiteSpace($URL) ? @{StatusCode = 400 } : $(Invoke-WebRequest -Uri $URL -Method Head -ErrorAction SilentlyContinue) # If the URL is null, return a status code of 400
+    # If the URL is null, return a status code of 400
+    if ([String]::IsNullOrWhiteSpace($URL)) {
+        $response = @{ StatusCode = 400 }
+    } else {
+        $response = Invoke-WebRequest -Uri $URL -Method Head -ErrorAction SilentlyContinue
+    }
     if ($response.StatusCode -ne 200) {
         Write-Debug "Fetching remote content from $URL returned status code $($response.StatusCode)"
         return $null
     }
-    $localFile = $OutputPath ? [System.IO.FileInfo]::new($OutputPath) : $(New-TemporaryFile) # If a path was specified, store it at that path; Otherwise use the temp folder
+    # If a path was specified, store it at that path; Otherwise use the temp folder
+    if ($OutputPath) {
+        $localFile = [System.IO.FileInfo]::new($OutputPath)
+    } else {
+        $localFile = New-TemporaryFile
+    }
     Write-Debug "Remote content will be stored at $($localFile.FullName)"
-    $script:CleanupPaths += $Raw ? @($localFile.FullName) : @() # Mark the file for cleanup when the script ends if the raw data was requested
+    # Mark the file for cleanup when the script ends if the raw data was requested
+    if ($Raw) {
+        $script:CleanupPaths += @($localFile.FullName)
+    } else {
+        $script:CleanupPaths += @()
```

Comment:
No need to add an empty item here, can simply exclude the else; Empty item was just for the ternary operator